addon/editor/input-handlers/disable-delete-handler.ts

```diff
@@ -13,6 +13,7 @@ export default class DisableDeleteHandler extends InputHandler {
   }
 
   handleEvent(_: KeyboardEvent): HandlerResponse {
+    this.rawEditor.model.read();
```

Comment:
this shouldn't be necessary, nothing is happening here?

Reply:
correct, wasn't paying enough attention